core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.ts

```diff
@@ -76,6 +68,47 @@ export class ContributorAdminDashboardPageComponent implements OnInit {
   checkMobileView(): boolean {
     return (this.windowRef.nativeWindow.innerWidth < 800);
   }
+
+  updateSelectedContributionType(selectedContributionType: string): void {
+    this.selectedContributionType = selectedContributionType;
+    this.setActiveTab(selectedContributionType);
+  }
+
+  ngOnInit(): void {
+    let userInfoPromise = this.userService.getUserInfoAsync();
+    userInfoPromise.then(userInfo => {
+      const username = userInfo.getUsername();
+      if (username) {
```

Comment:
Flip the condition using a guard clause to reduce nesting and increase readability.
```suggestion
      if (username === null) {
        return;
      }
     ...
```

Reply:
Sg. Done.